src/utils/data.ts

```diff
@@ -1,6 +1,8 @@
 export const merge = <T extends {}>(...objs: T[]): T =>
   Object.assign({}, ...objs)
 
-export function castArray<TValue>(value: TValue | TValue[]): TValue[] {
+export function castArray<TValue>(
+  value: TValue | ReadonlyArray<TValue>
+): ReadonlyArray<TValue> {
```

Comment:
We are returning a mutable array. So let's return `TValue[]`.

Reply:
Sorry! I will fix this soon.

Reply:
completed! https://github.com/thebrodmann/deox/pull/132/commits/5210e82f593533bb9ba57ae85e668384e93b3b44